AttitudeManager/Src/AM.cpp

```diff
@@ -1,3 +1,82 @@
 //
 // Created by Anthony Luo on 2022-10-12.
 //
+#include "AM.hpp"
+#include "LOS_Actuators.hpp"
+#include <cstdlib>
+#include <array>
+
+namespace AM {
+
+void AttitudeManager::runControlLoopIteration(
+    const AttitudeManagerInput &instructions) {
+    // Process Instructions
+
+    // Run Control Algorithms
+    std::vector<ActuatorOutput> controller_output;
+    if (current_controller_index == desired_controller_index) {
+        controller_output = controller_interfaces[current_controller_index]
+                                ->runControlsAlgorithm(instructions);
+    } else {
+        // Handle transition between flight modes
+        controller_output = runTransitionMixingIteration(
+            controller_interfaces[current_controller_index],
+            controller_interfaces[desired_controller_index], instructions);
+    }
+
+    // Write actuator outputs
+    LOS_Actuators outputs {};
+    for (auto output : controller_output) {
+         outputs.set(output.channel, output.percent);
```

Comment:
I think this will need to be updated to `Los_Actuators::getInstance().set(uint8_t channel, uint8_t percent);`